src/js/modules/wallet/controllers/send-confirm-modal/send-confirm-modal.tpl.html

```diff
@@ -74,7 +74,17 @@ <h3 class="modal-title">{{ 'SENDING_CONFIRM_TITLE' | translate }}</h3>
 
         <div ng-if="working && !complete">
             <div class="progress">
-                <div class="progress-bar" role="progressbar" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100" style="width: {{progressWidth}}%;"></div>
+                <div class="progress-bar progress-bar-striped active"
+                     ng-class="{
+                         'progress-bar-warning': walletData.networkType == 'BTC',
+                         'progress-bar-success': walletData.networkType != 'BTC'
+                      }"
```

Comment:
why only on BTC and not on BCH?

Reply:
warning is yellow and it fits well with the blue for BTC. Yellow is also for the progress bar, but we can go green for every send-progress bar as well.